routes/order/util.js

```diff
@@ -73,6 +73,33 @@ module.exports = {
       }
       await dynamo.update(updateMachineParams)
     }
+  },
+
+  async getItemInfo (itemId, dynamo) {
+    const itemCheckParams = {
+      TableName: 'items',
+      Key: {
+        item_id: itemId
+      },
+      AttributesToGet: ['volume', 'weight']
+    }
+
+    const itemCheckResponse = await dynamo.get(itemCheckParams)
+
+    if (!itemCheckResponse.Item) {
+      return null
+    }
+
+    const weight = itemCheckResponse.Item.weight
+    const volume = itemCheckResponse.Item.volume
+
+    const itemInfo = { 'itemWeight': weight, 'itemVolume': volume }
+    return itemInfo
+  },
+
+  createMachineOrder (machineOrder, itemID, itemQuantity, itemInfo, itemLocation) {
+    machineOrder[itemID] = {quantity: itemQuantity, weight: itemInfo['itemWeight'], volume: itemInfo['itemVolume'], row: itemLocation.row, column: itemLocation.column}
+    return machineOrder
```

Comment:
Potential issue here. If an item doesn't exist, `getItemInfo` returns null. That is fine, but in createMachineOrder, you do `itemInfo['itemWeight']`. If itemInfo is null, this will throw an uncaught error because the field doesn't exist in null. You should do a null check somewhere to make sure you don't try to access a field of a null value.

Please add a unit test for `createMachineOrder` that takes in a null itemInfo, so you can verify this fix